test/images/image-util.sh

```diff
@@ -29,7 +29,8 @@ declare -A QEMUARCHS=( ["amd64"]="x86_64" ["arm"]="arm" ["arm64"]="aarch64" ["pp
 
 # Returns list of all supported architectures from BASEIMAGE file
 listArchs() {
-  cut -d "=" -f 1 "${IMAGE}"/BASEIMAGE
+  image=$1
```

Comment:
Why is this variable rename needed (IMAGE -> image)?

Reply:
Oops, looks like you use it in the call to eval at line 167.

Reply:
I wonder if it's possible to avoid the variable rename by just wrapping image-util.sh with another script. Then you might be able to avoid this variable rename. Wdyt?

Reply:
Indeed, $IMAGE is $2 which is given to the script, while $image is precicely the image to build. Indeed, it is a bit confusing, and we should change this. Wrapping image-util.sh in another script sounds a bit bloat-y IMO.

Then, how about this? We change:

    IMAGE=$2

to

    WHAT=$2

It makes sense, as you call ``make WHAT=image-name`` when building the image, it makes it more consistent.

So, we can continue to still have $IMAGE as it was (we'd still have to have ``IMAGE=$1`` at the beginning of functions though). Although uppercase variables are meant to be for global variables... so, maybe keep ``image=$1``?

Reply:
Renamed ``$IMAGE`` to ``$WHAT`` for consistency with the Make file, and left ``$image`` to refer strictly to the image to build / push (not uppercase, because it was made a local variable; not a global anymore).